bot/kodiak/queue.py

```diff
@@ -315,7 +316,7 @@ async def enqueue_for_repo(
 
 
 def get_merge_queue_name(event: WebhookEvent) -> str:
```

Comment:
We start a queue workers per queue name, so by updating this queue name to be per target, we'll get queues per target branch.

Reply:
Could we try and get the old version and the new version at the same time?

Reply:
I'm not sure what you mean.

Reply:
I was thinking we'd check for existence of the old queues, before creating the new ones

Not sure if that would work though, because when would the repo get switched to the new queue?